### PR TITLE
formatting source to fit 120 char width

### LINKED ISSUE #537: Linting setting
`"max-line-length": [true, 120]` should be the correct setting. It currently generates a bunch of errors. Will fix some and exclude some.

### PR DESCRIPTION
**Pull request by HerringtonDarkholme**
I also explicitly exclude some file like css-schema / htmlTags.

Fix #537

### CODE REVIEW

**Comment by octref** [comment]
Maybe revert the changes in the ignored file?
Actually I think it's better to put `/* tslint:disable:max-line-length */` on top of each of the files you want to ignore. Somehow tslint's exclude is not working for me:

image

File: tslint.json [hunk 1]
```diff
@@ -1,12 +1,11 @@
 {
   "rules": {
     "class-name": true,
-    "curly": true,
-    "import-spacing": true,
+    "curly": true, "import-spacing": true,
```
Comment (octref) [inline tslint.json]: Why same line? Maybe format this json file with prettier too.

File: package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "compile": "tsc -p client",
     "precommit": "npm run lint",
     "prepush": "npm run test",
-    "lint": "tslint 'server/src/**/*.ts' 'client/src/**/*.ts'",
+    "lint": "tslint --project tslint.json",
```
Comment (octref) [inline package.json]: tslint's --project points to tsconfig.json. You need to use -c here. But then we don't have a top-level tsconfig...

Comment (HerringtonDarkholme) [reply package.json]: -c only specify configuration file, but not input file. Only specify configuration will make tslint complain no input file. 

Strangely, tslint 5.8.0 accepts tslint.json as project file.

**Comment by octref** [comment]
Looks good, thanks!

**Pull request merged**